modules/images-referencing-images-imagestreams.adoc

```diff
@@ -0,0 +1,175 @@
+// Module included in the following assemblies:
+// * openshift_images/tagging-images
+
+[id="images-referencing-images-imagestreams_{context}"]
+= Referencing images in imagestreams
+
+You can use tags to reference images in imagestreams using the following
+reference types.
+
+.Imagestream reference types
+[width="50%",options="header"]
+|===
+|Reference type |Description
+
+|`ImageStreamTag`
+|An `ImageStreamTag` is used to reference or retrieve an image for a given
+imagestream and tag.
+
+|`ImageStreamImage`
+|An `ImageStreamImage` is used to reference or retrieve an image for a given
+imagestream and image name.
```

Comment:
An `ImageStreamImage` is used to reference or retrieve an image for a given
imagestream and sha id ?